pages.pt_PT/common/cd.md

```diff
@@ -0,0 +1,19 @@
+# cd
+
+> Mudar o diretório atual.
```

Comment:
Perhaps we should add the link to the official docs here (https://docs.microsoft.com/windows-server/administration/windows-commands/cd.)

Reply:
This is the `cd` command in the `common` directory. `cd` functions differently on different platforms, so I'm not sure it's suitable to link to the Windows documentation here.

Reply:
@schneiderl Should I do anything here or is the problem solved? Should I add the link or not?

Reply:
@FilipaDurao do not add the link. @sbrl comment is on point, we should not add something system specific to the common directory.